src/Microsoft.DotNet.Configurer/AspNetCertificateSentinel.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using Microsoft.DotNet.Cli.Utils;
+using Microsoft.Extensions.EnvironmentAbstractions;
+
+namespace Microsoft.DotNet.Configurer
+{
+    public class AspNetCertificateSentinel : IAspNetCertificateSentinel
```

Comment:
Not for you, but at some point we should unify these Sentinel interfaces. I don't think really need multiple of these interfaces just to convey that they are a different file. But don't worry about it in this PR.